server/blob/blob-api/pom.xml

```diff
@@ -54,7 +54,6 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <scope>test</scope>
```

Comment:
Why?

Reply:
We use it in `DeduplicatingBlobStoreImpl`

Reply:
Is it ok to have Impl in the same maven module than APIs ?

That's good to me but I want to ask it out loud...

Reply:
It's ok for me, let's ask the more opinionated team members @mbaechler ? :smile: